pkg/operator/starter.go

```diff
@@ -211,5 +305,5 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	<-ctx.Done()
 
-	return fmt.Errorf("stopped")
+	return nil
```

Comment:
It seems this last chunk is extra (part of https://github.com/openshift/openstack-cinder-csi-driver-operator/pull/155 that was not backported to 4.15).

Reply:
Done